packages/slider/src/spectrum-config.js

```diff
@@ -51,6 +69,24 @@ const config = {
                     ],
                     'variant'
                 ),
+                {
+                    find: [
+                        builder.class('spectrum-Slider-labelContainer'),
+                        builder.combinator('+'),
+                        builder.class('spectrum-Slider-controls'),
+                    ],
+                    replace: [
+                        {
+                            replace: builder.id('label-container'),
+                        },
+                        {
+                            replace: builder.combinator('+'),
+                        },
+                        {
+                            replace: builder.id('track'),
```

Comment:
Can this change come from spectrum-css? My only goal here is to keep spectrum-config as light as possible

Reply:
I'll make sure to get this in the conversation with their team:
1. ticks vs tick labels rules is 100% a bug on their end (but it is breaking our users, and causing them to leverage tools like `patch-pachage` to work around, so fast forwarding until they've been able to fix is important)
2. we've discussed the `<output>` delivery before, they called it "correct", so I'm not sure we'll get tracking there
3. the use of the `#track` element is something we added to complete the functional contract of the track. The actual CSS element don't go all the way to the left/right edges of the element, which means they have less fidelity in the pointerdown/move/up events than is needed
4. it's not in this PR but it's always a good idea to remind them that the ticks and the min/max position for the handle do not line up with the current visual delivery.
While they cycle on their end, I think we should move forward with these changes. We can revert/match to their final decisions later, but right now there are low quality visual deliveries hitting visitors today that we should clear up.